.github/workflows/cypress_tests.yaml

```diff
@@ -50,25 +51,37 @@ jobs:
                     # build assets
                     npm run build
                     sudo chmod -R 777 config/ public/files/ public/theme/ public/thumbs/ var/
+                    
+                    #Configure PHP to disable Session garbage collector and fix the timezone to UTC like Bolt use to compare date.
+                    echo "date.timezone=UTC" >> php.ini
+                    echo "session.gc_probability=0" >> php.ini
```

Comment:
Maybe better to use https://github.com/shivammathur/setup-php?tab=readme-ov-file#ini-values-optional instead?